python/ray/tune/integration/lightgbm.py

```diff
@@ -1,7 +1,88 @@
-from ray.train.lightgbm import (  # noqa: F401
-    RayTrainReportCallback as TuneReportCheckpointCallback,
-)
-from ray.util.annotations import Deprecated
+import tempfile
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Dict, Optional
+
+from lightgbm.basic import Booster
```

Comment:
does this also work?

```suggestion
from lightgbm import Booster
```

Reply:
yep, changed.